rules/scoped-css/rule.md

```diff
@@ -0,0 +1,118 @@
+---
+type: rule
+title: Do you know when to use scoped CSS?
+uri: scoped-css
+authors:
+  - title: Jack Reimers
+    url: https://www.ssw.com.au/people/jack-reimers
+  - title: Harry Ross
+    url: https://www.ssw.com.au/people/harry-ross
+  - title: Matt Wicks
+    url: https://www.ssw.com.au/people/matt-wicks
+created: 2023-09-29T12:00:00.000Z
+guid: 16300344-d1b7-4dbe-abfa-e8b18d8f0c8d
+---
+
+On large frontend projects with lots of components it's common to have issues with your CSS classes overwriting each other or conflicting. 
+There are a few different ways to solve this.
+
```

Comment:
maybe add a picture that demonstrates the pain
a div that says I am blue (but it is red)
underneath I can see a css file with 3k+ lines (or something else that would suggest that it would be hard to debug)